server/queue/queue-rabbitmq/src/main/java/org/apache/james/queue/rabbitmq/RabbitMQManagementApi.java

```diff
@@ -58,6 +69,8 @@
             .logLevel(Logger.Level.FULL)
             .encoder(new JacksonEncoder())
             .decoder(new JacksonDecoder())
+            .retryer(new Retryer.Default())
```

Comment:
the new retry retries 5 times by default. Is it expected?

Reply:
Yes. Why not? 5 is better than 1, no? It's not exactly what I did previously but it does solve the exposed problem with minimal complexity in our code, so I went for it.

Reply:
5 is better than 1? I don't know, it depends on the context :)